.travis.yml

```diff
@@ -2,6 +2,9 @@ language: generic
 services: docker
 
 script:
+  - >
+    docker run -v=$(pwd):/app -v=/tmp/coala:/cache
+    --workdir=/app coala/base coala --ci
```

Comment:
is it possible it indent the parameter so it doesnt align with the command?

Reply:
Did you mean 
```
- >
    docker run -v=$(pwd):/app -v=/tmp/coala:/cache
      --workdir=/app coala/base coala --ci
```

Reply:
@aashutoshrathi Without the unnecessary `-v=/tmp/coala:/cache`, this command can be written as a one-liner